extensions/amp-date-picker/0.1/amp-date-picker.css

```diff
@@ -15,3 +15,7 @@
  */
 
 @import '../../../node_modules/react-dates/lib/css/_datepicker.css';
+
+.DateInput_input {
+  font-size: 100%;
```

Comment:
Prevents iOS from zooming in when tapping on the hidden input. The default was `11px`. Anything less than `16px` triggers auto-zooming on focus.

Reply:
wow, good to know!